### PR TITLE
MM-1734 Changed language in error message and did cosmetic refactoring

### PR DESCRIPTION
**Pull request by nickago**
Cosmetic refactoring includes:
   + Double quotes to single quotes
   + Remove unwanted spaces

### CODE REVIEW

File: web/react/components/signup_team_complete.jsx [hunk 1]
```diff
@@ -496,60 +496,58 @@ SendInivtesPage = React.createClass({
...
 
         var username_error = utils.isValidUsername(name);
-        if (username_error === "Cannot use a reserved word as a username.") {
-            this.setState({name_error: "This username is reserved, please choose a new one." });
+        if (username_error === 'Cannot use a reserved word as a username.') {
+            this.setState({name_error: 'This username is reserved, please choose a new one.'});
             return;
         } else if (username_error) {
-            this.setState({name_error: "Username must begin with a letter, and contain between 3 to 15 lowercase characters made up of numbers, letters, and the symbols '.', '-' and '_'." });
+            this.setState({name_error: "Username must begin with a letter, and contain between 3 to 15 characters made up of numbers, lowercase letters, and the symbols '.', '-', and '_'"});
```
Comment (crspeller) [inline web/react/components/signup_team_complete.jsx]: "...characters made up of numbers..."? Is this the text we want here? Characters are not made up of numbers.

How about: "Username must begin with a later, and contain 3 to 15 additional characters that may be numbers, lowercase letters, or any of the symbols '.', '-', or '_'"

Comment (nickago) [inline web/react/components/signup_team_complete.jsx]: This is the text that the ticket wanted specifically, so I would want to check with @it33 if this works better, but then yah that's fine.

Comment (crspeller) [inline web/react/components/signup_team_complete.jsx]: Waiting for @it33 to confirm this is the text we want.

Comment (it33) [inline web/react/components/signup_team_complete.jsx]: @nickago, agree with @crspeller on the issue he noted. Propose: 

Username must begin with a letter, and contain 3 to 15 characters in total, which may be numbers, lowercase letters, or any of the symbols '.', '-', or '_'

**Comment by nickago** [comment]
@crspeller Updated with @it33's message

**Pull request merged**